cmd/run.go

```diff
@@ -429,15 +432,434 @@ func init() {
 	RootCmd.AddCommand(RunCmd)
 }
 
+func executeRun(runFilePath string, apps []runfileconfig.App) (bool, error) {
+	var exitWithError bool
+
+	// setup shutdown notify channel.
+	sigCh := make(chan os.Signal, 1)
+	setupShutdownNotify(sigCh)
+
+	runStates := make([]*runExec.RunExec, 0, len(apps))
+	for _, app := range apps {
+		// Set defaults if zero value provided in config yaml.
+		app.RunConfig.SetDefaultFromSchema()
+
+		// Validate validates the configs and modifies the ports to free ports, appId etc.
+		err := app.RunConfig.Validate()
+		if err != nil {
+			print.FailureStatusEvent(os.Stderr, "Error validating run config for app %s present in %s: %s", app.RunConfig.AppID, runFilePath, err.Error())
+			exitWithError = true
+			break
+		}
+
+		// Get Run Config for different apps.
+		runConfig := app.RunConfig
+
+		appLogWriter, err := app.SetAndGetAppLogWriter()
+		if err != nil {
+			print.StatusEvent(os.Stderr, print.LogFailure, "Error getting log file for app %s present in %s: %s", runConfig.AppID, runFilePath, err.Error())
+			exitWithError = true
+			break
+		}
+		daprdLogWriter, err := app.SetAndGetDaprdLogWriter()
+		if err != nil {
+			print.StatusEvent(os.Stderr, print.LogFailure, "Error getting log file for app %s present in %s: %s", runConfig.AppID, runFilePath, err.Error())
+			exitWithError = true
+			break
+		}
+		// create a multiwriter to write to both app and daprd log files.
+		appDaprdWriter := io.MultiWriter(appLogWriter, daprdLogWriter)
+		runState, err := startDaprdAndAppProcesses(&runConfig, app.AppDirPath, sigCh, daprdLogWriter, daprdLogWriter, appLogWriter, appLogWriter)
+		if err != nil {
+			print.StatusEvent(os.Stdout, print.LogFailure, "Error starting Dapr and app (%q): %s", app.AppID, err.Error())
+			print.StatusEvent(appDaprdWriter, print.LogFailure, "Error starting Dapr and app (%q): %s", app.AppID, err.Error())
```

Comment:
can we not include os.Stdout also in `io.MultiWriter` above ?

Reply:
`os.Stdout` cannot be included in multi writer as `os.Stdout` sometimes has slightly different behavior ie adding color etc.